test/rules/deprecation/test.ts.lint

```diff
@@ -75,7 +76,6 @@ fn<number>();
 fn(1);
 ~~ [err % ('fn')]
 fn;
-~~ [err % ('fn')]
```

Comment:
This line no longer triggers a linter error, which is intended. `fn` has multiple overloads and not all of them are deprecated.